app/command.go

```diff
@@ -241,11 +252,12 @@ func (a *App) mentionsToTeamMembers(message, teamId string) model.UserMentionMap
 	}
 
 	atMentionMap := make(model.UserMentionMap)
```

Comment:
nit: maybe define this after the `wg.Wait` so as to couple closer with the for loop where it's populated?

Reply:
I agree that it is more readable defining it just before the loop, but I decided to put it before the `wg.Wait()` to maximize concurrency. Not really important, though, the performance change must be as negligible as it can be.

Reply:
Done in https://github.com/mattermost/mattermost-server/pull/13762/commits/9c01498c3cde0eac0ad1882bbf9f66459083f19b.